securing_apps/topics/overview/terminology.adoc

```diff
@@ -0,0 +1,14 @@
+=== Terminology
+
+These terms are used in this guide:
+
+* `Clients` are entities that can request {project_name} to authenticate a user. Most often, clients are applications and services that want to use {project_name} to secure themselves and provide a single sign-on solution. Clients can also be entities that just want to request identity information or an access token so that they can securely invoke other services on the network that are secured by {project_name}.
```

Comment:
Perhaps: "Clients are entities that interact with {project_name} to authenticate users and obtain tokens. Most often, clients are applications and services acting on behalf of users that provide a Single Sign-On experience to their users and access other services using the tokens issued by the server. Clients can also be entities only interested in obtaining tokens and acting on their own behalf for accessing other services."